tests/rdma/Makefile

```
@@ -0,0 +1,16 @@
+BIN = rdma-test
+
+ifeq ($(RDMA_PKGCONFIG),0)
+        RDMA_LIBS=$(shell $(PKG_CONFIG) --libs librdmacm libibverbs)
+else
+        RDMA_LIBS=-lrdmacm -libverbs
+endif
+
+binary:
+	@$(CC) rdma-test.c -o $(BIN) $(RDMA_LIBS) -lpthread -g
```

If we use the binary file name as the target name, then make can understand if it needs to be built or if nothing needs to be done.

```suggestion
$(BIN): rdma-test.c
	@$(CC) @< -o $@ $(RDMA_LIBS) -lpthread -g
```

---

> If we use the binary file name as the target name, then make can understand if it needs to be built or if nothing needs to be done.

A typo in this command, it should be `@$(CC) $^ -o $@ $(RDMA_LIBS) -lpthread -g`.
Thanks a lot!